test-blink.cpp

```diff
@@ -32,19 +35,49 @@ int main(void)
         return 1;
     }
 
-    wiringPiSetupGpio();
-    pinMode(gpio, OUTPUT);
+    // Setup GPIO with libgpiod
+    struct gpiod_chip *chip;
+
+    chip = gpiod_chip_open("/dev/gpiochip0");
+    if (!chip)
+    {
+        std::cerr << "Failed to open gpiochip: /dev/gpiochip0" << std::endl;
+        return 1;
+    }
+
+    line = gpiod_chip_get_line(chip, gpio);
+    if (!line)
+    {
+        std::cerr << "Failed to get line! Output code: " << envGPIO << std::endl;
+        return 1;
+    }
+
+    int ret = gpiod_line_request_output(line, CONSUMER, 0);
+    if (ret < 0)
+    {
+        std::cerr << "Request line as output failed! Output code: " << ret << std::endl;
+        return 1;
+    }
 
     for (;;)
     {
-        digitalWrite(gpio, HIGH);
+        setLineValue(line, 1);
         std::cout << "On" << std::endl;
-        delay(500);
+        usleep(5000);
```

Comment:
This is changing 500ms to 5ms!

Reply:
I've changed to 100ms but not committed yet, is this delay good enough?

Reply:
Changing it affects the wiki and documentation. Why does it need to change and blink so fast?

Reply:
You're right, I'm going to leave it as 500ms.